pyproject.toml

```diff
@@ -19,13 +19,14 @@ classifiers = [
 ]
 
 dependencies = [
-    "numpy~=2.1.0",
+    "numpy~=1.25",
```

Comment:
Change back to 2.1.0 and create a fluid prop copy in the repository.

Reply:
done